src/pages/index.js

```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
+      <div className="cols-50 rows-fr-auto pl-4 pr-4 grid grid-cols-2 grid-rows-2 gap-4">
+        <div className="col-span-2 flex content-center justify-center md:col-span-1 md:row-span-2">
+          <img
+            src="/static/cover-1-4b42b6386c67ac1a1074e200b0daf0a4.jpg"
+            alt="ALT"
+          />
+        </div>
+        <div className="bg-gray-300 p-16 col-span-2 md:col-span-1 md:row-span-2 flex flex-col content-center justify-center">
+          <h3 className="uppercase text-4xl leading-tight tracking-tight mb-4 md:text-2xl lg:text-5xl">
+            Enhance and <i className="lowercase">restore</i> your{" "}
+            <i className="lowercase">natural</i> beauty.
+          </h3>
+          <p className="mb-4 text-gray-900 md:text-xs lg:text-sm">
+            Dr Yalda is an expert in identifying the treatments that will help
+            you look and feel your best. Her experience and artistic eye in
+            conjunction with using the most premium brands allows her to provide
+            her patients with satisfied, long lasting results:
+          </p>
+          <ul>
+            {/* <li className="pb-1 pt-1">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                className="inline mr-2 text-gray-900"
+              >
+                <polyline points="20 6 9 17 4 12"></polyline>
+              </svg>
+              <p className="text-gray-900 inline md:text-xs lg:text-sm">
+                Reducing the signs of ageing
               </p>
-              <ul>
-                <li className="pb-1 pt-1">
-                  <Check className="inline mr-2 text-gray-900" />
-                  <p className="text-gray-900 inline">
-                    Reducing the signs of ageing
-                  </p>
-                </li>
-                <li className="pb-1 pt-1">
-                  {" "}
-                  <Check className="inline mr-2 text-gray-900" />
-                  <p className="text-gray-900 inline">
-                    Defining your facial features
-                  </p>
-                </li>
-                <li className="pb-1 pt-1">
-                  {" "}
-                  <Check className="inline mr-2 text-gray-900" />
-                  <p className="text-gray-900 inline">
-                    Improving your overall skin health
-                  </p>
-                </li>
-              </ul>
-            </div>
-          </div>
+            </li>
+            <li className="pb-1 pt-1">
+              {" "}
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
```

Comment:
Remove all this. save this in a text file on your laptop if you want it later

Reply:
Can't see this either